lib/omnibus/cli.rb

```diff
@@ -66,7 +66,9 @@ def build(name)
       project = Project.load(name)
 
       say("Building #{project.name} #{project.build_version}...")
-      project.build_me
+      manifest = project.resolve_manifest
+      project.download(manifest)
```

Comment:
It feels like a project instance should just expose a `#manifest` instance method and use that internally in the `#download` instance method.